docs/map-making/how-to-upgrade-legacy-map-features.md

```diff
@@ -88,21 +56,21 @@ _(these items were already deprecated and did nothing)_
 - `<option name="giveUnitControl" value="true"/>`
 - `<option name="giveUnitControl" value="false"/>`
 
-### Changes of map (properties) file items:
+## map.properties Changes
 
 Within every "map" properties file, inside the main folder of the map, change:
 
 - `color.Impassible=` >> `color.Impassable=`
 
-### Changes of sounds (properties) file items:
+## Sound file changes
 
-Within every "sounds" properties file, inside the main folder of the map, change:
+Change all sounds inside the "sounds" main folder of the map from "wav" to "mp3" format.
+128Kbps and 168Kbps are the preferred bit rates.
 
-- `.wav` >> `.mp3`
 
-### Changes of non-code items:
+## More XML Changes
```

Comment:
Since, as explained, all changes that follow are not additional changes, but merely "unnecessary additional information", I believe this "More XML Changes" entry is confusing or even wrongly descriptive (since from this point onwards there are no more changes of any kind, just alternative ones and additional information).